src/dg/dg.cpp

```diff
@@ -581,28 +583,46 @@ void DGBase<dim,real,MeshType>::assemble_cell_residual (
 
     const dealii::types::global_dof_index current_cell_index = current_cell->active_cell_index();
 
-    assemble_volume_term_explicit (
-        current_cell,
-        current_cell_index,
-        fe_values_volume,
-        current_dofs_indices,
-        current_cell_rhs,
-        fe_values_lagrange);
-    current_cell_rhs*=0.0;
-    //if ( compute_dRdW || compute_dRdX || compute_d2R ) {
+    const unsigned int grid_degree = this->high_order_grid->fe_system.tensor_degree();
+    const unsigned int poly_degree = i_fele;
+  //  const unsigned int grid_degree = current_metric_cell->active_fe_index();
+
+    if(!this->all_parameters->use_weak_form 
+       && this->all_parameters->ode_solver_param.ode_solver_type == Parameters::ODESolverParam::ODESolverEnum::explicit_solver )//only for strong form explicit
```

Comment:
Could declare `using ODE_enum = Parameters::ODESolverParam::ODESolverEnum` at the top of this function and replace this line with `&& this->all_parameters->ode_solver_param.ode_solver_type == ODE_enum::explicit_solver)` to clean it up

Reply:
fixed